src/SFA.DAS.LevyTransferMatching/InnerApi/LevyTransferMatching/Requests/CreatePledgeRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using SFA.DAS.LevyTransferMatching.Models;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.LevyTransferMatching.InnerApi.LevyTransferMatching.Requests
+{
+    public class CreatePledgeRequest : IPostApiRequest
+    {
+        private readonly long _accountId;
+
+        public long AccountId => _accountId;
```

Comment:
Other than for testing, I can't see where this property is used - is it needed?

Reply:
I think it was just there for the test, rather than having to examine the route

Reply:
May be redundant - I'll look to remove it